Mods/ArchipelagoPlayer.SC2Mod/enUS.SC2Data/LocalizedData/GameHotkeys.txt

```diff
@@ -29,6 +29,7 @@ Button/Hotkey/AP_BuildAutoTurret=T
 Button/Hotkey/AP_BuildCreepTumor=C
 Button/Hotkey/AP_BuildCreepTumorPropagate=C
 Button/Hotkey/AP_BuildCyclone=N
+Button/Hotkey/AP_BuildDevastatorTurret=G

```

Comment:
What about using `V` as a hotkey?

Reply:
I set it to V initially, then when I went to test realized the Swann hotkey is G and I was pressing G out of habit, so set it back to that.